android/src/main/java/com/google/android/cameraview/Camera1.java

```diff
@@ -706,12 +718,12 @@ private int calcDisplayOrientation(int screenOrientationDegrees) {
      * @return Number of degrees to rotate image in order for it to view correctly.
      */
     private int calcCameraRotation(int screenOrientationDegrees) {
-        if (mCameraInfo.facing == Camera.CameraInfo.CAMERA_FACING_FRONT) {
-            return (mCameraInfo.orientation + screenOrientationDegrees) % 360;
-        } else {  // back-facing
-            final int landscapeFlip = isLandscape(screenOrientationDegrees) ? 180 : 0;
-            return (mCameraInfo.orientation + screenOrientationDegrees + landscapeFlip) % 360;
-        }
+       if (mCameraInfo.facing == Camera.CameraInfo.CAMERA_FACING_FRONT) {
+           return (mCameraInfo.orientation + screenOrientationDegrees) % 360;
```

Comment:
?why front facing camera does not need landspaceFlip/

Reply:
Idk why but changing only applying it to the front facing instead of the back facing fixes this issue 🤣 At least on my Nexus 5X...